Source/Core/VideoCommon/TextureConversionShader.cpp

```diff
@@ -950,6 +950,7 @@ uint GetTiledTexelOffset(uint2 block_size, uint2 coords)
   return buffer_pos;
 }
 
+#if defined(HAS_PALETTE)
```

Comment:
It seems a bit odd to use defines here, but conditionally write/don't write stuff in the shader source elsewhere.  (This isn't new; `#if defined(PALETTE_FORMAT_IA8)` and similar were already in use a few lines below.  I just find it a bit odd).